packages/vx-chord/src/Ribbon.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import cx from 'classnames';
+import { ribbon as d3ribbon, Chord, ChordSubgroup } from 'd3-chord';
+
+export type RibbonProps = {
+  chord: Chord;
+  source?: (d: Chord) => ChordSubgroup;
+  target?: (d: Chord) => ChordSubgroup;
+  radius?: (d: ChordSubgroup) => number;
+  startAngle?: (d: ChordSubgroup) => number;
+  endAngle?: (d: ChordSubgroup) => number;
+  children?: ({ path }: { path: string | null }) => string | undefined;
+  className?: string;
+};
+
+export default function Ribbon({
+  chord,
+  source,
+  target,
+  radius,
+  startAngle,
+  endAngle,
+  children,
+  className,
+  ...restProps
+}: RibbonProps & Omit<React.SVGProps<SVGPathElement>, keyof RibbonProps>) {
+  const ribbon = d3ribbon<any, Chord, ChordSubgroup>();
+  if (source) ribbon.source(source);
+  if (target) ribbon.target(target);
+  if (radius) ribbon.radius(radius);
+  if (startAngle) ribbon.startAngle(startAngle);
+  if (endAngle) ribbon.endAngle(endAngle);
+  const path = ribbon(chord) as any;
```

Comment:
can this not be any? based on other `d3` `path` generators, it should return a `string` or `null`? 

[the types seem to suggest this](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/d3-chord/index.d.ts#L269) as well.

Reply:
TypeScript not inferring the correct method overload in this case :( Not sure what is the best way to fix this